### PR TITLE
[izumi-veizi] add strategy for calculating votes according to amount of veiZi

### PR DESCRIPTION
**Pull request by LpcAries**
Fixes # .

Changes proposed in this pull request:
- add new strategy for calculating votes according to amount of veiZi
- 
-

### CODE REVIEW

File: src/strategies/izumi-veizi/index.ts [hunk 1]
```diff
@@ -0,0 +1,79 @@
+import { BigNumberish } from '@ethersproject/bignumber';
+import { formatUnits } from '@ethersproject/units';
+import { Multicaller } from '../../utils';
+import abi from './abi.json'
```
Comment (ChaituVR) [inline src/strategies/izumi-veizi/index.ts:4]: It is not required to pass the entire json, just pass the abi of the method you are using :)

File: src/strategies/izumi-veizi/index.ts [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+export const author = 'izumi';
```
Comment (ChaituVR) [inline src/strategies/izumi-veizi/index.ts:6]:
```suggestion
export const author = 'izumiFinance';
```

File: src/strategies/izumi-veizi/index.ts [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+                const decimalPoint = Number(Math.floor(parseFloat(formatUnits(point, options.decimals)) * 100) / 100)
+                const tmp = Object.keys(ans).find((t)=>{return t===address})
+                if (tmp !== undefined) {
+                    ans[address] = ans[address] + decimalPoint
+                } else {
+                    Object.assign(ans, {[address]:decimalPoint})
+                }
+          })   
+        }   
+    }))
```
Comment (ChaituVR) [inline src/strategies/izumi-veizi/index.ts:51]: It should not call the contract once per address. https://github.com/snapshot-labs/snapshot-strategies#checklist-for-adding-a-new-strategy

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/strategies/izumi-veizi/index.ts (hunk 3 above)
Comment (LpcAries) [reply src/strategies/izumi-veizi/index.ts]: Code has been updated, waiting for your reply

**Pull request merged**